File: CPP Programs/Exp 1.cpp
// C++ Program to find whether the given number is palindrome or not?

#include<iostream>
using namespace std;
int main()
{
int n,reverse=0,rem;
cin>>n;
  while(n!=0)
  {
      rem=n%10;
      reverse=reverse*10+rem;
      n/=10;
  }
  cout<<""<<reverse<<" is a palindrome"<<endl;
  return 0;
    
}

File: Python Programs/Exp 1.py
# Python Program to find whether the given number is palindrome or not?

n=1001
temp=n
rev=0
while(n>0):
    dig=n%10
    rev=rev*10+dig
    n=n//10
if(temp==rev):
    print("1001 is a palindrome")
else:
    print("The number isnt a palindrome")